pkg/pool-weighted/contracts/lbp/LiquidityBootstrappingPool.sol

```diff
@@ -61,21 +73,27 @@ contract LiquidityBootstrappingPool is BaseWeightedPool, ReentrancyGuard {
     // For gas optimization, store start/end weights and timestamps in one bytes32
     // Start weights need to be high precision, since restarting the update resets them to "spot"
     // values. Target end weights do not need as much precision.
-    // [     32 bits   |     32 bits     |      64 bits     |      124 bits      |    3 bits    |     1 bit    ]
-    // [ end timestamp | start timestamp | 4x16 end weights | 4x31 start weights |   not used   | swap enabled ]
-    // |MSB                                                                                                 LSB|
+    // [     32 bits   |     32 bits     |      64 bits     |      124 bits      |  2 bits  |  1 bit   |     1 bit    ]
+    // [ end timestamp | start timestamp | 4x16 end weights | 4x31 start weights | not used | recovery | swap enabled ]
+    // |MSB                                                                                                        LSB|
 
     bytes32 private _poolState;
 
     // Offsets for data elements in _poolState
     uint256 private constant _SWAP_ENABLED_OFFSET = 0;
-    uint256 private constant _START_WEIGHT_OFFSET = 4;
-    uint256 private constant _END_WEIGHT_OFFSET = 128;
-    uint256 private constant _START_TIME_OFFSET = 192;
-    uint256 private constant _END_TIME_OFFSET = 224;
+    uint256 private constant _RECOVERY_MODE_BIT_OFFSET = 1;
+    uint256 private constant _START_WEIGHT_OFFSET = _RECOVERY_MODE_BIT_OFFSET + 2;
```

Comment:
```suggestion
    uint256 private constant _START_WEIGHT_OFFSET = _RECOVERY_MODE_BIT_OFFSET + 3;
```

We haven't moved the start weights so they should still start 4 bits in.

Reply:
Ha! Good catch, though it would still work (overwriting one of the unused bits)